inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -227,21 +241,43 @@ else if (annotation instanceof Sentence) {
                                 // Cast to Object so that the proper valueOf signature is used by
                                 // the compiler, otherwise it will think that a String argument is
                                 // char[].
-                                featureValue = String.valueOf((Object) WebAnnoCasUtil
+                              
+                            	featureValue = String.valueOf((Object) WebAnnoCasUtil
                                         .getFeature(annotation, feature.getName()));
                                 
+                                String labelStr = null;
+                                if(feature.getUiName().equals("identifier") && featureValue!="null") 
+                                	 labelStr = getUILabel(featureValue);
+                               
                                 // Add the UI annotation.feature name to the index as an annotation.
                                 // Replace spaces with underscore in the UI name.
                                 
-                                MtasToken mtasAnnotationFeature = new MtasTokenString(mtasId++,
+                                MtasToken mtasAnnotationFeatureIRI = new MtasTokenString(mtasId++,
                                         annotationUiName.replace(" ", "_") + "."
                                                 + feature.getUiName().replace(" ", "_")
                                                 + MtasToken.DELIMITER + featureValue,
                                         beginToken);
-                                mtasAnnotationFeature.setOffset(annotation.getBegin(),
+                                mtasAnnotationFeatureIRI.setOffset(annotation.getBegin(),
                                         annotation.getEnd());
-                                mtasAnnotationFeature.addPositionRange(beginToken, endToken);
-                                tokenCollection.add(mtasAnnotationFeature);
+                                mtasAnnotationFeatureIRI.addPositionRange(beginToken, endToken);
+                                tokenCollection.add(mtasAnnotationFeatureIRI);
+                                
+                                
+                                // Add the UI annotation.feature label to the index as an annotation.
+                                // Replace spaces with underscore in the UI name.
+                                if(labelStr!=null) {
```

Comment:
There should be  space between `if` and `(`. Please make sure that you have Checkstyle installed in your Eclipse and that it works. See https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins/job/INCEpTION%20(GitHub)%20(master)/de.tudarmstadt.ukp.inception.app$inception-app-webapp/doclinks/3/#_checkstyle